app/code/core/Mage/GoogleAnalytics/Helper/Data.php

```diff
@@ -104,4 +106,15 @@ public function isUseAnalytics4($store = null)
     {
         return Mage::getStoreConfig(self::XML_PATH_TYPE, $store) == self::TYPE_ANALYTICS4;
     }
+
+    /**
+     * Whether GA Enhanced eCommerce data should be submitted
+     *
+     * @param null|string|bool|int|Mage_Core_Model_Store $store $store
+     * @return bool
+     */
+    public function isEnhancedECommEnabled($store = null)
+    {
+        return Mage::getStoreConfigFlag(self::XML_PATH_ECOMM, $store);
+    }
```

Comment:
do we need to have the separate setting? I think maybe we could enable it by default, don't everybody want those data on GA4?

Reply:
We definitely want it 100% of the time, i was just trying to preserve the original behavior. In case someone didn't want?
Can definitely remove the checks and just always send the data?

Reply:
I'd remove it and have everything in the _getOrdersTrackingCodeAnalytics4 method, if would also allow for removing the modification to the system.xml file ;-)

Reply:
(and the translation)